rubocop_base_config.yml

```diff
@@ -122,7 +122,8 @@ Lint/UnexpectedBlockArity: # new in 1.5
 Lint/UnmodifiedReduceAccumulator: # new in 1.1
   Enabled: true  
 Lint/UselessMethodDefinition: # (new in 0.90)
-  Enabled: true
```

Comment:
I'm not sure if the `Enabled: true` is still needed here. According to ChatGPT it isn't 🤔

Reply:
I don't think it is, but it leaves a trace of what new rules were added between updates.